### DISCUSSION TITLE
Support for Streamdeck Plugins

### DISCUSSION THREAD

**Discussion by OmegaRogue** [streamdeck-linux-gui/streamdeck-linux-gui#62] [Ideas / Feature Requests]
Having support for the normal Streamdeck plugins would greatly improve this software, as it would allow alot more things without having to seperately add everything

**Comment by Data-007** [comment]
https://github.com/streamdeck-linux-gui/streamdeck-linux-gui/discussions/42

I would like this option to, I have not explained it very well. It basically would lend 
to our OWN plugins being made as basic scripts.  If your looking at having the windblows
plugins then I think that this will NOT happen.
Just my thoughts.

**Comment by RemcoSchrijver** [comment]
> Having support for the normal Streamdeck plugins would greatly improve this software, as it would allow alot more things without having to seperately add everything

@OmegaRogue Streamdeck plugins are written in JavaScript and with the intention to be ingested by the Elgato Streamdeck application. Maybe a translation layer can be built but on first look that seems difficult. 

I think it is important to first look at plugin support in general written specifically for this application before trying to explore this venture.
